components/esp_dns/include/esp_dns.h

```diff
@@ -0,0 +1,64 @@
+/*
+ * SPDX-FileCopyrightText: 2025 Espressif Systems (Shanghai) CO LTD
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#pragma once
+
+#include "sdkconfig.h"
+#include "esp_err.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#define DEFAULT_TCP_PORT 53 /* Default TCP port */
+#define DEFAULT_DOT_PORT 853 /* Default TLS port */
+#define DEFAULT_DOH_PORT 443 /* Default HTTPS port */
+
+#define DEFAULT_TIMEOUT_MS 5000
```

Comment:
suggest namespacing with `ESP_` or `ESP_DNS` (since this is a public header and `DEFAULT_TCP_PORT` is a just a common name)

Reply:
Done